django/utils/datetime_safe.py

```diff
@@ -1,3 +1,5 @@
+# RemovedInDjango50Warning
```

Comment:
Oh, I have seen comments used to flag non-obvious things to be removed related to a deprection but all these comments look redundant to me.

Reply:
Thanks. I'll remove the comments on review (as in these cases it would clearly blow up) but I (we) don't mind the comments in-general, even if maybe redundant in some cases. _Super obvious_ is good. It may be _me_ removing the code, and much better if I don't have to think 😉